.github/workflows/linux_meson.yml

```
@@ -102,3 +102,111 @@ jobs:
       run: |
         export OMP_NUM_THREADS=2
         python dev.py -n -j 2
+
+  prerelease_deps_coverage_64bit_blas:
+    name: Prerelease deps and Blas 64bit
+    # If using act to run CI locally the github object does not exist and the usual skipping should not be enforced
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Setup Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Ubuntu dependencies
+      run: |
+        # NOTE: not the same OpenBLAS version as in upstream CI (I'm being lazy here)
+        sudo apt-get update
+        sudo apt-get install -y libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
+
+    - name: Caching Python dependencies
+      uses: actions/cache@v3
+      id: cache
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip
+
+    - name: Install Python packages
+      run: |
+        python -m pip install "setuptools<60.0" wheel ninja meson==0.60.3
```

```suggestion
        python -m pip install setuptools wheel ninja meson==0.60.3
```

I don't think we need to pin setuptools anymore since this job uses meson (especially not setuptools<60.0.0 since that's been deprecated).

---

Why do we need `setuptools` and `wheel` at all?

---

Just trying to remove both.